### PR TITLE
Add docs for family providers

### PR DESCRIPTION
**Pull request by turkenf**
### Description of your changes

This PR adds documentation for family providers

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: docs/family/Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+```
+
+_Note_: official provider-families only support `up` command-line versions v0.13.0 or later.(CHECK HERE)
+
+## Install Universal Crossplane
+Install Upbound Universal Crossplane with the Up command-line.
+
+```shell
+$ up uxp install
+UXP 1.9.0-up.3 installed
```
Comment (jeanduplessis) [inline docs/family/Quickstart.md:34]: Lets update to latest version here.

Also let's add a note of the minimum version required for running family providers.

Comment (turkenf) [reply docs/family/Quickstart.md]: I updated and added a note.

File: docs/family/Quickstart.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+crossplane-7fdfbd897c-pmrml                 1/1     Running   0             68m
+crossplane-rbac-manager-7d6867bc4d-v7wpb    1/1     Running   0             68m
+upbound-bootstrapper-5f47977d54-t8kvk       1/1     Running   0             68m
+xgql-7c4b74c458-5bf2q                       1/1     Running   3 (67m ago)   68m
+```
+
+## Install the official GCP provider-family
+
+Install the official provider-family into the Kubernetes cluster with a Kubernetes configuration file.
+For instance, let's install the `provider-gcp-storage`
```
[the remark this answers is not in the document]
Comment (turkenf) [reply docs/family/Quickstart.md:49]: Done, thank you!

File: docs/monolith/Quickstart.md
Comment (sergenyalcin) [inline docs/monolith/Quickstart.md:1]: We may consider changing the line 159 with the following to be compatible with the family doc:

`Create a managed resource to verify the provider is functioning. ` => 

`Create a managed resource to verify the provider-gcp is functioning. `

File: docs/monolith/Configuration.md
[the remark this answers is not in the document]
Comment (turkenf) [reply docs/monolith/Configuration.md:1]: I changed the titles.

File: docs/monolith/Quickstart.md
[the remark this answers is not in the document]
Comment (turkenf) [reply docs/monolith/Quickstart.md:1]: Done.

[the remark this answers is not in the document]
Comment (turkenf) [reply docs/monolith/Quickstart.md:1]: Done.

[the remark this answers is not in the document]
Comment (turkenf) [reply docs/monolith/Quickstart.md:1]: I changed this line.

Comment (turkenf) [reply docs/monolith/Quickstart.md:1]: Done, thank you!

**Review by sergenyalcin — APPROVED** [review]
Thanks @turkenf LGTM!

**Review by jeanduplessis — APPROVED** [review]

**Pull request merged**